src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -468,3 +468,47 @@ void Instance::OnCommissioningComplete(CHIP_ERROR err)
 } // namespace Clusters
 } // namespace app
 } // namespace chip
+
+bool emberAfNetworkCommissioningClusterAddOrUpdateThreadNetworkCallback(
```

Comment:
Shouldn't we do something to not generate calls to these at all, since it's a waste of codesize?  Followup needed for that.

Reply:
Agree, but now there are still platforms which don't have full network commissioning driver support, and they need these functions. We could remove the generate calls after all the platforms add the updated network commissioning driver and the `network-commissioning-old` is removed.

Reply:
That is why I said "followup needed".